assets/company-admin/components/CompanyAdminApp.jsx

```diff
@@ -134,6 +134,9 @@ class CompanyAdminAppComponent extends React.Component {
                                     {gettext('Users')}
                                 </button>
                             </div>
+
+                            {this.props.sectionId === 'users' ? <label className='label label--big label--rounded'>USERS : {
```

Comment:
Use `gettext` to flag this label for translation (also include `totalusers` in the translation)